src/bootstrap/bin/rustc.rs

```diff
@@ -232,13 +232,8 @@ fn main() {
             }
         }
 
-        if let Ok(s) = env::var("RUSTC_CRT_STATIC") {
```

Comment:
Er, this commit seems to have been sort of truncated  in that I don't see any additions to builder.rs here?